packages/decap-cms-widget-markdown/src/MarkdownControl/plugins/html/withHtml.js

```diff
@@ -0,0 +1,98 @@
+import { jsx } from 'slate-hyperscript';
+import { Transforms } from 'slate';
+
+const ELEMENT_TAGS = {
+  A: el => ({ type: 'link', url: el.getAttribute('href') }),
+  BLOCKQUOTE: () => ({ type: 'quote' }),
+  H1: () => ({ type: 'heading-one' }),
+  H2: () => ({ type: 'heading-two' }),
+  H3: () => ({ type: 'heading-three' }),
+  H4: () => ({ type: 'heading-four' }),
+  H5: () => ({ type: 'heading-five' }),
+  H6: () => ({ type: 'heading-six' }),
+  IMG: el => ({ type: 'image', url: el.getAttribute('src') }),
+  LI: () => ({ type: 'list-item' }),
+  OL: () => ({ type: 'numbered-list' }),
+  P: () => ({ type: 'paragraph' }),
+  PRE: () => ({ type: 'code' }),
+  UL: () => ({ type: 'bulleted-list' }),
+};
+
+// COMPAT: `B` is omitted here because Google Docs uses `<b>` in weird ways.
```

Comment:
Except on line 29, `B` is, in fact, not omitted. Should the comment, or the `B` mapping be removed?

Reply:
great catch.
Slate also noticed and updated this, so I just copied new version of example.

> Should the comment, or the B mapping be removed?

Since it's known that Google Docs uses `<b>` differently, I would leave it removed as is